nbss/app.py

```diff
@@ -148,6 +148,8 @@ async def download(request: Request, notebook_id: str = ID_VALIDATOR):
             # Allow JupyterLab to fetch this URL from anywhere with just JS
             # Supports https://github.com/jupyterlab/jupyterlab/pull/11387
             "Access-Control-Allow-Origin": "*",
+            # Allow access to extra headers such as Content-Disposition
+            "Access-Control-Expose-Headers": "*",
```

Comment:
We could also limit this to:

```suggestion
            "Access-Control-Expose-Headers": "Content-Disposition",
```